### PR TITLE
I18n of Select

### PR DESCRIPTION
**Pull request by RaoHai**
i18n of select and demos

### CODE REVIEW

File: components/select/index.en-US.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+```html
+<Select>
+  <Option value="lucy">lucy</Option>
+</Select>
+```
+
+## API
+
+### Select props
```
Comment (warmhug) [inline components/select/index.en-US.md]: api 文档为什么不直接抄 rc 组件里的 英文文档？保持一致更新也方便的

Comment (RaoHai) [inline components/select/index.en-US.md]: 好建议……

**Comment by superRaytin** [comment]
#1471

File: components/select/demo/optgroup.md [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
+  zh-CN: 分组
+  en-US: Option Group
 ---
 
+## zh-CN
 用 `OptGroup` 进行选项分组。
 
+## en-US
+
+using `OptGroup` to group the options.
```
Comment (waywardmonkeys) [inline components/select/demo/optgroup.md]: Capitalize 'Using' here.

File: components/select/demo/search-box.md [hunk 3]
```diff
@@ -1,10 +1,17 @@
...
+  zh-CN: 搜索框
+  en-US: Search Box
 ---
 
+## zh-CN
 带有搜索按钮的自动补全输入框。
 
+## en-US
+
+Autocomplete select with search field
```
Comment (waywardmonkeys) [inline components/select/demo/search-box.md]: Missing period at end of the sentence.

File: components/select/index.en-US.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+category: Components
+type: Form Control
+title: Select
+---
+
+A Selector similar to Select2.
+
+## When to use
+
+Select provides a dropdown menu for choosen, for replacing origin selector, or demanding of a elegent one.
```
Comment (waywardmonkeys) [inline components/select/index.en-US.md]: Not sure what is meant here by 'demanding' ... and spelling error on 'elegant'.

File: components/select/demo/coordinate.md [hunk 5]
```diff
@@ -1,12 +1,20 @@
...
+  en-US: coordinate
 ---
 
+## zh-CN
 省市联动是典型的例子。
 
 推荐使用 [Cascader](/components/cascader) 组件。
 
+## en-US
+Coordination of Province and Cities is a classic demostration.
```
Comment (waywardmonkeys) [inline components/select/demo/coordinate.md]: Coordinating the selection of provinces and cities is a common use case and demonstrates how selection can be coordinated.

... that still sounds awkward.

File: components/select/demo/coordinate.md [hunk 6]
```diff
@@ -1,12 +1,20 @@
...
+
+[Cascader](/components/cascader) component is strongly recommend.
```
Comment (waywardmonkeys) [inline components/select/demo/coordinate.md]: Using the [Cascader](/components/cascader) component is strongly recommended instead as it is more flexible and capable.

(Is it more flexible and capable?)

File: components/select/demo/label-in-value.md [hunk 7]
```diff
@@ -1,12 +1,20 @@
...
+## zh-CN
 默认情况下 `onChange` 里只能拿到 value，如果需要拿到选中的节点文本 label，可以使用 `labelInValue` 属性。
 
 选中项的 label 会被包装到 value 中传递给 `onChange` 等函数，此时 value 是一个对象。
 
+## en-US
+As a default behavior, `onChange` callback can only get value of selected item. For getting label property of selected item, you can use `labelInValue` props.
+
+Label of selected item will be packed as an object for passing to `onChange` callback.
+
```
Comment (waywardmonkeys) [inline components/select/demo/label-in-value.md]: As a default behavior, the `onChange` callback can only get the value of the selected item. The `labelInValue` prop can be used to get the label property of the selected item.

The label of the selected item will be packed as an object for passing to the `onChange` callback.

(... what does that mean, packed as an object?)

File: components/select/demo/multiple.md [hunk 8]
```diff
@@ -1,10 +1,17 @@
...
+  zh-CN: 多选
+  en-US: mutiple selection
 ---
 
+## zh-CN
 多选，从已有条目中选择（scroll the menu）
 
+## en-US
+
+Multiple selection, select from exist items (scroll the menu).
```
Comment (waywardmonkeys) [inline components/select/demo/multiple.md]: Multiple selection, selecting from existing items (scroll the menu).

File: components/select/demo/size.md [hunk 9]
```diff
@@ -1,10 +1,18 @@
...
+  en-US: three sizes
 ---
 
+## zh-CN
+
 三种大小的选择框，当 size 分别为 `large` 和 `small` 时，输入框高度为 `32px` 和 `22px` ，默认高度为 `28px`
 
+## en-US
+
+Select of three sizes. The height of input field will be `32px` when `size` get `large`, `22px` when `small`, and default to `28px`.
```
Comment (waywardmonkeys) [inline components/select/demo/size.md]: The height of the inpub field for the select defaults to `28px`. If `size` is set to `large`, the height will be `32px`, and if set to `small`, `22px`.

File: components/select/index.en-US.md (hunk 4 above)
Comment (afc163) [inline components/select/index.en-US.md]: `A dropdown menu for choosen, a elegent alternate for native select comopnent.`

Comment (waywardmonkeys) [inline components/select/index.en-US.md]: A dropdown menu for choosing, an elegant alternative to the native select component.

Comment (afc163) [inline components/select/index.en-US.md]: 👍

File: components/select/demo/coordinate.md [hunk 10]
```diff
@@ -1,12 +1,20 @@
...
+  zh-CN: 联动
+  en-US: coordinate
 ---
 
+## zh-CN
 省市联动是典型的例子。
 
 推荐使用 [Cascader](/components/cascader) 组件。
 
+## en-US
```
Comment (afc163) [inline components/select/demo/coordinate.md]: Add a blank line below this.

File: components/select/demo/coordinate.md [hunk 11]
```diff
@@ -1,12 +1,20 @@
 ---
 order: 6
-title: 联动
+title: 
+  zh-CN: 联动
+  en-US: coordinate
 ---
 
+## zh-CN
```
Comment (afc163) [inline components/select/demo/coordinate.md]: Add a blank line below this.

File: components/select/demo/multiple.md [hunk 12]
```diff
@@ -1,10 +1,17 @@
 ---
 order: 2
-title: 多选
+title: 
+  zh-CN: 多选
+  en-US: mutiple selection
 ---
 
+## zh-CN
 多选，从已有条目中选择（scroll the menu）
```
Comment (afc163) [inline components/select/demo/multiple.md]: Add a blank line below.

Other titles below should be fixed too.

File: components/select/demo/size.md [hunk 13]
```diff
@@ -1,10 +1,18 @@
 ---
 order: 0
-title: 三种大小
+title: 
+  zh-CN: 三种大小
+  en-US: three sizes
```
Comment (afc163) [inline components/select/demo/size.md]: Three sizes

File: components/select/demo/label-in-value.md (hunk 7 above)
Comment (afc163) [inline components/select/demo/label-in-value.md]: `label` of selected item will be inserted to augument `value` of `onChange` callback function.

File: components/select/demo/coordinate.md (hunk 6 above)
Comment (afc163) [inline components/select/demo/coordinate.md]: more friendly in interaction

**Comment by benjycui** [comment]
@waywardmonkeys  you are back, thanks! 😀

**Comment by RaoHai** [comment]
@waywardmonkeys thanks ~

File: components/select/demo/optgroup.md [hunk 14]
```diff
@@ -1,10 +1,17 @@
 ---
 order: 5
-title: 分组
+title: 
+  zh-CN: 分组
+  en-US: Option Group
 ---
 
+## zh-CN
 用 `OptGroup` 进行选项分组。
```
Comment (afc163) [inline components/select/demo/optgroup.md]: Add a blank line below.

File: components/select/demo/search-box.md [hunk 15]
```diff
@@ -1,10 +1,17 @@
 ---
 order: 9
-title: 搜索框
+title: 
+  zh-CN: 搜索框
+  en-US: Search Box
 ---
 
+## zh-CN
 带有搜索按钮的自动补全输入框。
```
Comment (afc163) [inline components/select/demo/search-box.md]: Add a blank line below.

File: components/select/demo/size.md [hunk 16]
```diff
@@ -11,7 +11,7 @@ title:
 
 ## en-US
 
-Select of three sizes. The height of input field will be `32px` when `size` get `large`, `22px` when `small`, and default to `28px`.
+The height of the inpub field for the select defaults to 28px. If size is set to large, the height will be 32px, and if set to small, 22px.
```
Comment (ddcat1115) [inline components/select/demo/size.md]: The height of the __input__ field...

Comment (afc163) [inline components/select/demo/size.md]: 1. remove all extra `the`
2. 

```
If size is set to large, the height will be 32px, and if set to small, 22px.

=>

`32px` for size `large` and `22px` for size `small`.
```

**Comment by benjycui** [comment]
@RaoHai PR 根据 review 更新下？

**Comment by RaoHai** [comment]
我就 rebase 了一下 master， 为什么那么多提交……

**Comment by afc163** [comment]
感觉 rebase 错了。

**Comment by RaoHai** [comment]
重新 rebase 然后 push -f 了一下，应该好了。

**Pull request merged**